projects/packages/forms/tests/php/contact-form/Contact_Form_Test.php

```diff
@@ -887,18 +866,30 @@ public function test_make_sure_checkbox_field_renders_as_expected() {
 
 	/**
 	 * Multiple fields
-	 *
-	 * @covers Automattic\Jetpack\Forms\ContactForm\Contact_Form_Field
 	 */
 	public function test_make_sure_checkbox_multiple_field_renders_as_expected() {
 		$attributes = array(
-			'label'   => 'fun',
-			'type'    => 'checkbox-multiple',
-			'class'   => 'lalala',
-			'default' => 'option 1',
-			'id'      => 'funID',
-			'options' => array( 'option 1', 'option 2' ),
-			'values'  => array( 'option 1', 'option 2' ),
+			'label'       => 'fun',
+			'type'        => 'checkbox-multiple',
+			'class'       => 'lalala',
+			'default'     => 'option 1',
+			'id'          => 'funID',
+			'options'     => array( 'option 1', 'option 2' ),
+			'values'      => array( 'option 1', 'option 2' ),
+			'optionsdata' => wp_json_encode(
```

Comment:
Use optionsdata to presume that `block_attributes_to_shortcode_attributes` has taken place. Let's rather test that as a unit.